### DISCUSSION TITLE
CD Changer

### DISCUSSION THREAD

**Discussion by u038265** [BlueSCSI/BlueSCSI-v2#220] [General]
Hello,
I installed a BlueSCSI Desktop V2 on my Apple Macintosh SE.
The computer configuration is: Italian System 7.1, 4MB RAM and SuperDrive.
the bluescsi.ini file is the following:

[SCSI]
System="Generic"
Debug=0; Off
SelectionDelay=255
Quirks=1 ; Apple
EnableUnitAttention=0 ; Off
EnableSCSI2=1 ; On
EnableSelLatch=0 ; Off
EnableParity=1 ; On
MapLunsToIDs=0 ; Off
MaxSyncSpeed=10
ReinsertCDOnInquiry=0 ; Off
DisableROMDrive=1 ; Disabled
ROMDriveSCSIID=2
InitiatorMode=0; Off
DisableConfigHook=1
WiFiSSID=MyWIFI
WiFiPassword=WIFIPassword

[SCSI0]
Vendor=BlueSCSI
Product=DesktopHD
Version=2
Serial=0123456789ABCDEF
Type=0 ; Fixed
IMG0=HD00_512  1024 MB System.hda
SectorsPerTrack=63
HeadsPerCylinder=255
BlockSize=512
RightAlignStrings=1
PrefetchBytes=8192

[SCSI1]
Vendor=BlueSCSI
Product=DesktopHD
Version=2
Serial=0123456789ABCDEF
Type=0 ; Fixed
IMG0=HD10_512 BlueSCSI Toolbox 1-0-1.hda
SectorsPerTrack=63
HeadsPerCylinder=255
BlockSize=512
RightAlignStrings=1
PrefetchBytes=8192

[SCSI2]
Vendor=BlueSCSI
Product=DesktopHD
Version=2
Serial=0123456789ABCDEF
Type=0 ; Fixed
IMG0=HD20_512 BlueSCSI Bootstrap.hda
SectorsPerTrack=63
HeadsPerCylinder=255
BlockSize=512
RightAlignStrings=1
PrefetchBytes=8192

[SCSI3]
Vendor=BlueSCSI
Product=DesktopCD
Version=2
Serial=0123456789ABCDEF
Type=2 ; Optical
BlockSize=2048
RightAlignStrings=1
PrefetchBytes=8192

In the HD0 directory I put the file 'HD00_512  1024 MB System.hda'
In the HD1 directory I put the file 'HD10_512 BlueSCSI Toolbox 1-0-1.hda'
In the HD2 directory I put the file 'HD20_512 BlueSCSI Bootstrap.hda'
In the CD3 directory I put three working ISO files.
In the root directory there is the empty text file 'NE4.hda' for Ethernet network access.
When I run the 'CD Changer' program, an error window appears with the following message "No compatible BlueSCSI's found...".
I can't figure out how to solve the problem, can you help me?
Thanks in advance.

Massimo Massarenti
![20241214_111646](https://github.com/user-attachments/assets/be04d6ae-f280-4535-b54a-14ba31964d66)

**Comment by erichelgeson** [comment]
Please remove the bluescsi.ini file - 99% of computers do not need any configuration, should work great then.

**Comment by u038265** [comment]
Ok

Thank you.

**Comment by u038265** [comment]
Thanks Eric, everything works perfectly now.
